tracks/haskell/exercises/pangram/mentoring.md

```diff
@@ -4,19 +4,64 @@
 import Data.Char (toLower)
 
 isPangram :: String -> Bool
-isPangram xs = all (`elem` map toLower xs) ['a'..'z']
+isPangram s = all (`elem` map toLower s) ['a'..'z']
 ```
 
+This solution traverses the input once for each letter, but terminates once each was found.
```

Comment:
You could consider creating separate headings for each of the solutions, which can help in quickly scanning the reasonable solutions. I've done this in the C# and F# tracks, see for example [this file](https://github.com/exercism/website-copy/blob/b78072b386b9792b3d43cf4292d8d1dd1b4b6712/tracks/csharp/exercises/bob/mentoring.md).

Note: this is merely a suggestion

Reply:
> Thanks, I'll do that! Perhaps this suggestion should be added here.

This might be a good idea. Maybe @iHiD has any thoughts on whether these sub-headings for different reasonable solutions is a good idea?

Reply:
Yeah - I'm good with that as long as they're `####` and the general headings are respected. A PR to the readme would be very welcome :)